docs/core/compatibility/sdk/7.0/solution-level-output-no-longer-valid.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This should be added to two locations in https://github.com/dotnet/docs/blob/main/docs/core/compatibility/toc.yml.